sdk/eventgrid/azure-eventgrid/CHANGELOG.md

```diff
@@ -1,22 +1,40 @@
 # Release History
 
-## 4.19.1 (Unreleased)
+## 4.20.0 (Unreleased)
 
 ### Features Added
-
 ### Breaking Changes
+### Bugs Fixed
```

Comment:
Prob need to update this section/copy over the info from the beta entries below?

Reply:
yes was going to add after merge to main